pkg/config/xds/filter_types.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package xds
+
+// Import all Envoy filter types so they are registered and deserialization does not fail
+// when using them in the "typed_config" attributes.
```

Comment:
Ah. So this is why people got an error when they used typed_config in their envoy filters! . Thanks for the fix..